config/main.py

```diff
@@ -328,6 +328,9 @@ def run_command(command, display_cmd=False, ignore_error=False):
     if display_cmd == True:
         click.echo(click.style("Running command: ", fg='cyan') + click.style(command, fg='green'))
 
+    if os.environ["UTILITIES_UNIT_TESTING"] == "1":
+        return
```

Comment:
Is is best to add this here? Should we instead mock this function in any tests which should override this function?

Reply:
let me try that.